src/renderer/mod.rs

```diff
@@ -829,10 +829,11 @@ impl QuadRenderer {
                         error!("Error reading shader file: {}", err);
                     }
                     ShaderCreationError::Compile(path, log) => {
-                        error!("Error compiling shader at {:?}\n{}", path, log);
+                        error!("Error compiling shader at `{:?}`\n```\n{}\n```",
+                               path, log);
```

Comment:
Errors should just get printed normally, to be consistent everywhere. It seems to me like `Error compiling shader at ``{:?}``: {}` would be the correct choice here.